docs/products/opensearch/howto/connect-with-curl.rst

```diff
@@ -0,0 +1,131 @@
+Use Aiven for OpenSearch with cURL
+==================================
+
+Connect to your Aiven for OpenSearch service with `cURL <https://curl.se/>`_.
+
+Variables
+---------
+
+These are the placeholders you will need to replace in the code samples:
+
+==================      =============================================================
+Variable                Description
+==================      =============================================================
+``OPENSEARCH_URI``      Service URI, including username and password, available from
+                        the service overview page
+==================      =============================================================
+
+Connect to OpenSearch
+---------------------
+
+Connect to your service with::
+
+    curl OPENSEARCH_URI
+
+If the connection is successful, one of the nodes in your cluster will respond with some information including:
+
+* ``name`` The node name you are connected to (this will be the name of your service, with the node identifier suffix)
+
+* ``version`` The version information includes both the ``distribution`` and the ``version`` of the distribution that is running
+
+Manage indexes
+--------------
+
+OpenSearch groups data into an index rather than a table.
+
+Create an index
+'''''''''''''''
+
+Create an index by making a ``PUT`` call to it::
+
+    curl -X PUT OPENSEARCH_URI/shopping-list
```

Comment:
Should we include an example with mappings / settings?

Reply:
Do you have an example handy? I think that might be nice to have

Reply:
> Do you have an example handy? I think that might be nice to have

Yes, sure

```
curl -X PUT -H "Content-Type: application/json" \
    OPENSEARCH_URI/shopping-list \
    -d '{
        "mappings": {
            "properties": {
                "item": { "type": "keyword"  },
                "quantity": { "type": "integer" }
            }
        }
     }'
```

Reply:
I added some words as well, hope it makes sense!